homework/password-check/validation.cpp

```diff
@@ -1,2 +1,60 @@
 #include "validation.hpp"
-// TODO: Put implementations here
\ No newline at end of file
+// TODO: Put implementations here
+bool isspecial(char ch){
+    if(isalpha(ch)){
+		return false;
+    }
+	if(isdigit(ch)){
+		return false;
+    }
+	else{
+		return true;
+    }
```

Comment:
When you use return inside `if` you don't need `if else` or `else`. Just write `if (sth) { return blabla; }`